### PR TITLE
logo-cloud2

### PR DESCRIPTION
**Pull request by akhnos**

### CODE REVIEW

File: editor-components/logo-clouds/logo-comp2/logo-comp2.module.scss [hunk 1]
```diff
@@ -1,316 +1,62 @@
...
-                            height: 100%;
-
-                        }
-
-                    }
+            .image-child{
+                display: inline-block;
+                padding: var(--composer-gap-md) var(--composer-gap-lg);
+                background: color-mix(in srgb, var(--composer-html-background), var(--composer-font-color-primary) 5%);
+                border-radius: 200px;
```
Comment (Ayazelar) [inline editor-components/logo-clouds/logo-comp2/logo-comp2.module.scss:43]: We need to use css varible for border radius

File: editor-components/logo-clouds/logo-comp2/logo-comp2.tsx [hunk 2]
```diff
@@ -1,155 +1,126 @@
...
+          >
+            {this.getPropValue("description")}
+          </Base.SectionDescription>
+        </Base.VerticalContent>
+        <div className={this.decorateCSS("gallery")}>
+          <div
+            className={this.decorateCSS("images-container")}
+            style={{ animationDuration: `${images.length * 2}s` }}
+          >
+            {images.map((imageItem: any, index: number) => (
```
Comment (Ayazelar) [inline editor-components/logo-clouds/logo-comp2/logo-comp2.tsx:88]:
```suggestion
            {images.map((imageItem: TImage, index: number) => (
```

File: editor-components/logo-clouds/logo-comp2/logo-comp2.tsx [hunk 3]
```diff
@@ -1,155 +1,126 @@
...
+              </ComposerLink>
+            ))}
+          </div>
+          <div
+            className={`${this.decorateCSS(
+              "images-container"
+            )} ${this.decorateCSS("reverse")}`}
+            style={{ animationDuration: `${images.length * 2}s` }}
+          >
+            {images.map((imageItem: any, index: number) => (
```
Comment (Ayazelar) [inline editor-components/logo-clouds/logo-comp2/logo-comp2.tsx:107]:
```suggestion
            {images.map((imageItem: TImage, index: number) => (
```

**Review by Ayazelar — CHANGES_REQUESTED** [review]
.

File: editor-components/logo-clouds/logo-comp2/logo-comp2.module.scss (hunk 1 above)
Comment (akhnos) [reply editor-components/logo-clouds/logo-comp2/logo-comp2.module.scss:43]: There is not enough border-radius variable for this view. The maximum border-radius is less than 200px. For these cases, it is okay to send a specific border radius as in the example.

**Review by Ayazelar — APPROVED** [review]

**Pull request merged**